bin/cron/test_access_report_import

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/chat_client'
+
+CENSUS_BUCKET = 'cdo-census'.freeze
+
+def update_object_key(object_key, valid)
+  key_without_extension = object_key.chomp(".test")
+  AWS::S3.rename_object(CENSUS_BUCKET, object_key, key_without_extension + ".#{valid ? 'valid' : 'invalid'}")
```

Comment:
I just realized that the person running these tests might be carrying them out iteratively, so including a timestamp might be useful. For example: `/state_cs_offering/LA/2021-2022.test` --> `/state_cs_offering/LA/2021-2022.2001-02-03T04:05:06+03:00.invalid`